frontend/packages/jobs/src/trigger/jobs.ts

```diff
@@ -190,6 +168,7 @@ export const generateDocsSuggestionTask = task({
     installationId: number
     type: 'DOCS'
     branchName: string
+    overallReviewId?: number
```

Comment:
null should not exist. Please eliminate unnecessary existence checks too.

https://github.com/liam-hq/liam/pull/1302/files#diff-2058cacc0f2a83e932ddde6ece0f30766b197a7f2d42b3506be73b24adf87c4cR197